CooperationHullMainSite/wwwroot/css/site.css

```diff
@@ -318,12 +320,6 @@ h1 {
     padding: 4rem
 }
 
-@media screen and (min-width: 600px) {
-    .homepage-block > .row {
-        padding: 4rem 5rem
-    }
-}
-
```

Comment:
Removed this media query as it was a bit too tight on mobile. perhaps this work on the text-only sections? In which case it might make sense to extract into a narrower class that can be composed, but removing for now to elicit some comments :)